blueprints/ember-frost-object-browser/index.js

```diff
@@ -11,8 +11,8 @@ module.exports = {
   afterInstall: function () {
     return this.addAddonsToProject({
       packages: [
-        {name: 'ember-frost-core', target: '>=0.0.14 <2.0.0'},
-        {name: 'ember-frost-bunsen', target: '^3.0.0'},
+        {name: 'ember-frost-core', target: '>=0.1.0 <2.0.0'},
```

Comment:
I'm not sure if it's necessary, I just saw that when I installed `ember-frost-bunsen` it's `blueprints` added `0.1.x` to my deps